src/server/game/Chat/ChatLink.cpp

```diff
@@ -661,7 +661,7 @@ bool LinkExtractor::IsValidMessage()
             break;
 
         char commandChar;
-        _iss >> commandChar;
```

Comment:
There should be no need to delete

Reply:
_iss >> commandChar is the same to _iss.get(commandChar) ?